tests/test_setup_migrate_order.py

```
@@ -0,0 +1,130 @@
+        statusline_command=None,
+        statusline_global=False,
+        no_auto_install=True,
+    )
+    buf = io.StringIO()
+    # _cmd_setup may bail or print to stderr; we only care about ordering.
+    try:
+        cli_mod._cmd_setup(args, buf)
+    except Exception:
+        # Even if a downstream stub doesn't perfectly match the signature
+        # cli.py expects, the migration call should have happened first.
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)